src/components/video.js

```diff
@@ -47,47 +40,37 @@ const Video = () => {
   }
 
   // Mute all of the local user's tracks.
-  function muteAudio() {
+  const muteAudio = () => {
```

Comment:
Nice modularization. Keep this style for future additions to the video component.